File: Virucodesoup/Ruby/Standard/OFB (Output Feedback)/driver.rb
# ===========================
# File : driver.rb
#
# Credit:
# - Coded by (aka in cyber) Vsio Stitched
#
# Misc:
# - Written in Ruby programming language
#
# License:
# - Free to use
# - May include me or not in credit if included in other project
# ===========================

require_relative "ofb"

print "\n===== OFB (Output Feedback) =====\n"
print "\n\n== START PROGRAM ==\n\n"

ofbDriver = OFB.new
ofbDriver.driver()

print "\n\n== END PROGRAM ==\n\n"

gets

File: Virucodesoup/Javascript/External/Crafty/Draw Image/driver.js
// ===========================
// File : driver.js
//
// Credit:
// - Coded by (aka in cyber) Vsio Stitched
//
// Misc:
// - Written in Javascript programming language
// - Included Extension: Crafty
// 
// License:
// - Free to use
// - May include me or not in credit if included in other project
// ===========================

var SCREEN_WIDTH = 480;
var SCREEN_HEIGHT = 240;

window.onload = function() {

	Crafty.init(SCREEN_WIDTH, SCREEN_HEIGHT);
	Crafty.canvas.init();
	Crafty.background("aqua");
	
	DrawImage_Driver();
	
};